Makefile

```
@@ -24,7 +24,7 @@ IMAGE_PULL_POLICY ?= Always
 lint: lint-go-mod lint-modified-files
 
 lint-go-mod:
-	go mod verify
+	go mod tidy
 
 lint-modified-files:
```

We probably want to make sure we do the mod check before we check for modifications, using an [order-only prerequisites](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html):
```suggestion
lint-modified-files: | lint-go-mod
```

… But please actually check that this works as I expect, since I'm not 💯% certain about it.

---

This is correct.